app/css/index.css

```
@@ -93,12 +91,29 @@ body {
 	color: transparent;
 }
 
-.status {
+.status-red {
 	font-size: 24px;
 	text-wrap: balance;
 	text-align: center;
+	color: rgb(180, 60, 66);
+	border: 4px solid rgb(180, 60, 66);
```

just for consistency across (.status-green uses hex), use either hex value or RGB.
```suggestion
	border: 4px solid #b43c42;
```